crates/matrix-sdk/src/room/timeline/event_handler.rs

```diff
@@ -382,8 +385,11 @@ impl<'a> TimelineEventHandler<'a> {
                 }
             };
 
+            let mut msgtype = replacement.new_content;
+            msgtype.sanitize(DEFAULT_SANITIZER_MODE, RemoveReplyFallback::No);
```

Comment:
Maybe put the same `Edit's content is never supposed to contain the reply fallback.` comment here?